cmd/client/main_windows.go

```diff
@@ -0,0 +1,82 @@
+//go:build windows
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"syscall"
+
+	"github.com/NHAS/reverse_ssh/internal/client"
+)
+
+func printHelp() {
+
+	fmt.Println("usage: ", filepath.Base(os.Args[0]), "[--foreground] [--fingerprint] destination")
```

Comment:
note i didn't add detach to this help - its not really supposed to be manually specified. a bit wonky, but not sure a better way to communicate with the new child process